.travis.yml

```diff
@@ -16,8 +16,9 @@ before_install:
     - cd ..
     # - echo 'Skip Installing ci.common lib ....'
     - echo 'Installing ci.common lib ....'
-    - git clone https://github.com/OpenLiberty/ci.common.git ./ci.common
+    - git clone https://github.com/cherylking/ci.common.git ./ci.common
     - cd ./ci.common
+    - git checkout passDirectoryProperties
```

Comment:
When will this be reverted?  Same question for appveyor.yml

Reply:
I'll revert the yml changes before merging. Thanks for the reminder.